storage/s3/src/test/java/io/aiven/kafka/tieredstorage/storage/s3/S3MultiPartOutputStreamTest.java

```diff
@@ -231,9 +231,41 @@ void writesTailMessages() throws Exception {
 
         verify(mockedS3).initiateMultipartUpload(any(InitiateMultipartUploadRequest.class));
         verify(mockedS3, times(2)).uploadPart(any(UploadPartRequest.class));
-        verify(mockedS3).completeMultipartUpload(completeMultipartUploadRequestCaptor.capture());
+        verify(mockedS3, times(1)).completeMultipartUpload(completeMultipartUploadRequestCaptor.capture());
         assertCompleteMultipartUploadRequest(completeMultipartUploadRequestCaptor.getValue(),
             List.of(new PartETag(1, "SOME_ETAG#1"), new PartETag(2, "SOME_ETAG#2")));
+
+        assertStreamIsClosed(out);
+    }
+
+    @Test
+    void sendAbortIfNoWritingHappened() throws IOException {
+        when(mockedS3.initiateMultipartUpload(any(InitiateMultipartUploadRequest.class)))
```

Comment:
Nit: any()?

Reply:
Got caught up in the rebase, good catch, fixing it.